### PR TITLE
style: update texts and designs

### LINKED ISSUE #80: [SUGGESTION]: Review text in app
- [ ]  ["CollAction is an application powered by the crowd the more the better"](https://github.com/CollActionteam/collaction_app/blob/development/lib/presentation/crowd_action/pages/widgets/share_collaction_card.dart#L55)  
Should be _"the more, the better"_  
(Also, it seems some punctuation or parenthesis would make it easier to read: _". The more, [...] ."_ or _"(the more, [...] )"_)

### PR DESCRIPTION
**Pull request by yasht01**
- [x] User Profile page is yet to be edited (**_done_**).
- Updated tests for `contact screen` and `home screen` to the latest UX copy

Closes #80

### CODE REVIEW

File: lib/presentation/crowd_action/pages/crowd_action_details.dart [hunk 1]
```diff
@@ -204,7 +204,7 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
                     Padding(
                       padding: const EdgeInsets.fromLTRB(0, 10, 0, 10),
                       child: Text(
-                        'Short description about what the commitments are and how you can select/deselect them',
+                        'Your challenge, your rules.\n Choose which commitment you want to make this month.',
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/crowd_action_details.dart:207]: Remove space after `\n`

File: lib/presentation/crowd_action/pages/crowd_action_details.dart [hunk 2]
```diff
@@ -524,16 +462,17 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
                       height: 30,
                     ),
                     Text(
-                      widget.crowdAction.description,
+                      "You’re almost there! You’ve selected the displayed commitment to stick to through this CrowdAction. By clicking “Go!” you will officially commit to this goal.",
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/crowd_action_details.dart:465]: If more than 1 commitment is selected, we should show
"the displayed commitments"
"officially commit to these goals"
Plural forms.

File: lib/presentation/crowd_action/pages/crowd_action_details.dart [hunk 3]
```diff
@@ -1100,7 +1050,7 @@ class _BadgesPopupCard extends StatelessWidget {
                           maxWidth: 300,
                         ),
                         child: Text(
-                          'Short description about how you can earn different achievements/badges',
+                          'Different commitments give you a different set of points. These points correspond with badges.\n Which one will you attain this month?',
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/crowd_action_details.dart:1053]: Remove space after `\n`

File: lib/presentation/crowd_action/pages/crowd_action_details.dart [hunk 4]
```diff
@@ -1214,20 +1164,20 @@ class _BadgesPopupCard extends StatelessWidget {
...
-                      child: IconButton(
-                        onPressed: () => context.router.pop(),
-                        icon: Image.asset('assets/images/icons/close_icon.png'),
-                        color: Colors.pink,
-                      ),
+                ),
+                Align(
+                  alignment: Alignment.topRight,
+                  child: CircleAvatar(
+                    // color: kSecondaryColor,
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/crowd_action_details.dart:1171]: Remove comment

File: test/screens/home_screen_test.dart [hunk 5]
```diff
@@ -23,8 +24,12 @@ void main() {
       await tester.pumpWidget(AppWidget());
       await tester.pumpAndSettle();
 
+      // Get the current month
+      final date = DateTime.parse(DateTime.now().toString());
+      final month = DateFormat.MMMM().format(date);
```
Comment (Xazin) [inline test/screens/home_screen_test.dart:29]: `final month = DateFormat.MMMM().format(DateTime.now());`

File: test/screens/home_screen_test.dart [hunk 6]
```diff
@@ -23,8 +24,12 @@ void main() {
       await tester.pumpWidget(AppWidget());
       await tester.pumpAndSettle();
 
+      // Get the current month
+      final date = DateTime.parse(DateTime.now().toString());
```
Comment (Xazin) [inline test/screens/home_screen_test.dart:28]: Why are you parsing DateTime.now to DateTime.now ??

File: lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart [hunk 7]
```diff
@@ -134,4 +135,12 @@ class _InSpotLightHeaderState extends State<InSpotLightHeader> {
       ),
     );
   }
+
+  //? Move to Spotlight Bloc and fetch on loading the page?
+  String sectionHeadingText() {
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart:140]: ```String sectionHeadingText() => DateFormat.MMMM().format(DateTime.now());```

File: lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart [hunk 8]
```diff
@@ -134,4 +135,12 @@ class _InSpotLightHeaderState extends State<InSpotLightHeader> {
...
+    final _date = DateTime.parse(DateTime.now().toString());
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart:141]: ...

File: lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart [hunk 9]
```diff
@@ -134,4 +135,12 @@ class _InSpotLightHeaderState extends State<InSpotLightHeader> {
       ),
     );
   }
+
+  //? Move to Spotlight Bloc and fetch on loading the page?
```
Comment (Xazin) [inline lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart:139]: No, text should be loaded before bloc is done.

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 1 above)
Comment (yasht01) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:207]: @Xazin Viewing the text on the app makes this look better in my opinion. That's why I added `\n` in the below code.

File: test/screens/home_screen_test.dart (hunk 5 above)
Comment (yasht01) [reply test/screens/home_screen_test.dart:29]: Oh! My mistake. I'll make the changes.

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 1 above)
Comment (yasht01) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:207]: image
image

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 3 above)
Comment (yasht01) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:1053]: image
image

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 1 above)
Comment (Xazin) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:207]: But there's no reason to have the space that comes immediately after `\n`!! Also, this text should say "Choose which commitments" with plural commitments.

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 3 above)
Comment (Xazin) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:1053]: Again, I'm not talking about removing `\n`, I'm talking about removing this ` ` that comes after the `\n`

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 1 above)
Comment (yasht01) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:207]: Oh 🤦🏼‍♂️ will make the necessary changes.

**Comment by Xazin** [comment]
I think a push is missing locally @yasht01 ? Right?

**Comment by yasht01** [comment]
- Fix #85 and #89
- Resolve suggested changes

File: lib/presentation/auth/pages/profile_photo.dart [hunk 10]
```diff
@@ -0,0 +1,141 @@
+import 'dart:io';
```
Comment (Xazin) [inline lib/presentation/auth/pages/profile_photo.dart:1]: Rebase might have gone wrong, this is now in `/auth/widgets/profile_photo.dart`

Comment (Xazin) [reply lib/presentation/auth/pages/profile_photo.dart:1]: I will fix it

**Review by Xazin — APPROVED** [review]

**Pull request merged**